app/routes/_gcn.circulars.$circularId/route.tsx

```diff
@@ -7,43 +7,69 @@
  */
 import type { DataFunctionArgs, HeadersFunction } from '@remix-run/node'
 import { json } from '@remix-run/node'
-import { Link, useLoaderData } from '@remix-run/react'
+import { Link, Outlet, useLoaderData } from '@remix-run/react'
 import { Button, ButtonGroup, Icon } from '@trussworks/react-uswds'
 
-import { get } from '../_gcn.circulars/circulars.server'
-import { PlainTextBody } from './Body'
-import { FrontMatter } from './FrontMatter'
-import { origin } from '~/lib/env.server'
+import { getUser } from '../_gcn._auth/user.server'
+import {
+  get,
+  getVersions,
+  moderatorGroup,
+} from '../_gcn.circulars/circulars.server'
+import { feature, origin } from '~/lib/env.server'
 import { getCanonicalUrlHeaders, pickHeaders } from '~/lib/headers.server'
 import { useSearchString } from '~/lib/utils'
+import { useFeature } from '~/root'
 import type { BreadcrumbHandle } from '~/root/Title'
 
 export const handle: BreadcrumbHandle<typeof loader> = {
   breadcrumb({ data }) {
     if (data) {
-      const { circularId, subject } = data
+      const {
+        result: { circularId, subject },
+      } = data
       return `${circularId}: ${subject}`
     }
   },
 }
 
-export async function loader({ params: { circularId } }: DataFunctionArgs) {
+export async function loader({
+  params: { circularId, version },
+  request,
+}: DataFunctionArgs) {
   if (!circularId)
     throw new Response('circularId must be defined', { status: 400 })
-  const result = await get(parseFloat(circularId))
-  return json(result, {
-    headers: getCanonicalUrlHeaders(
-      new URL(`/circulars/${circularId}`, origin)
-    ),
-  })
+  const result = await get(
+    parseFloat(circularId),
+    version ? parseFloat(version) : undefined
+  )
```

Comment:
Why does this route need to get the circular? The route `_gcn.circulars.$circularId.($version)` does that.

Reply:
I think it's just for the bibcode, but I can probably refactor this

Reply:
@lpsinger actually this has me wondering, will each version have a unique bibcode, or will that persist through the versions?

Reply:
I don't know. Why does that matter for the bibcode? The ADS link should be in the child route.

Reply:
Gotcha, I updated it to be in the child route